docs/network/interfaces_and_networks.md

```diff
@@ -583,55 +582,10 @@ documentation](https://github.com/k8snetworkplumbingwg/sriov-network-operator/bl
 > policies, make sure you define a pool of VF resources that uses
 > `deviceType: vfio-pci`.
 
-Once the operator is deployed, an [SriovNetworkNodePolicy
-](https://github.com/k8snetworkplumbingwg/sriov-network-operator#sriovnetworknodepolicy)
-must be provisioned, in which the list of SR-IOV devices to expose (with
-respective configurations) is defined.
-
-Please refer to the following `SriovNetworkNodePolicy` for an example:
-
-```yaml
-apiVersion: sriovnetwork.openshift.io/v1
-kind: SriovNetworkNodePolicy
-metadata:
-  name: policy-1
-  namespace: sriov-network-operator
-spec:
-  deviceType: vfio-pci
-  mtu: 9000
-  nicSelector:
-    pfNames:
-    - ens1f0
-  nodeSelector:
-    sriov: "true"
-  numVfs: 8
-  priority: 90
-  resourceName: sriov-nic
-```
-
-The policy above will configure the `SR-IOV` device plugin, allowing the
-PF named `ens1f0` to be exposed in the SRIOV capable nodes as a resource named
-`sriov-nic`.
 
 #### Start an SR-IOV VM
 
-Once all the SR-IOV components are deployed, it is needed to indicate how to
-configure the SR-IOV network. Refer to the following
-`SriovNetwork` for an example:
-
-```yaml
-apiVersion: sriovnetwork.openshift.io/v1
-kind: SriovNetwork
-metadata:
-  name: sriov-net
-  namespace: sriov-network-operator
-spec:
-  ipam: |
-    {}
-  networkNamespace: default
-  resourceName: sriov-nic
-  spoofChk: "off"
-```
+Once all the SR-IOV components are deployed, and a network named `sriov-net` is deployed with a network-attachment-definition 
```

Comment:
The SR-IOV operator automatically creates a NetworkAttachmentDefinition.
Since it was removed, do you intend to add an example or link to an example? (could be done in a follow-up).

Reply:
let's take a follow-up on this